test/e2e/next-test-lib/next-modes/next-deploy.ts

```diff
@@ -4,9 +4,20 @@ import fs from 'fs-extra'
 import { platform } from 'os'
 import { NextInstance } from './base'
 
+type DeployRes = {
```

Comment:
[dust] Rename this with the full name.

```suggestion
type DeployResponse = {
```